.github/workflows/ci.yml

```diff
@@ -1,9 +1,9 @@
 name: CI
 on:
   push:
-    branches: [ main ]
```

Comment:
Please do not reformat the file and only remove the corresponding environment variable.

Reply:
The reformatted files have been removed